static/app/views/profiling/landing/profileCharts.tsx

```diff
@@ -24,40 +24,41 @@ interface ProfileChartsProps {
 // We want p99 to be before p75 because echarts renders the series in order.
 // So if p75 is before p99, p99 will be rendered on top of p75 which will
 // cover it up.
-const SERIES_ORDER = ['count()', 'p99()', 'p75()'] as const;
+const SERIES_ORDER = ['count()', 'p99()', 'p95()', 'p75()'] as const;
 
-export function ProfileCharts({router, selection}: ProfileChartsProps) {
+export function ProfileCharts({query, router, selection}: ProfileChartsProps) {
   const theme = useTheme();
 
-  const profileStats = useProfileStats({
-    query: '', // TODO: This doesnt support the same filters
-    selection,
+  const profileStats = useProfileEventsStats({
+    query,
+    yAxes: SERIES_ORDER,
   });
+
   const series: Series[] = useMemo(() => {
-    if (profileStats.type !== 'resolved') {
+    if (profileStats.status !== 'success') {
       return [];
     }
 
     // the timestamps in the response is in seconds but echarts expects
     // a timestamp in milliseconds, so multiply by 1e3 to do the conversion
-    const timestamps = profileStats.data.timestamps.map(ts => ts * 1e3);
+    const timestamps = profileStats.data[0].timestamps.map(ts => ts * 1e3);
 
-    const allSeries = profileStats.data.data
-      .filter(rawData => SERIES_ORDER.indexOf(`${rawData.axis}()` as any) > -1)
+    const allSeries = profileStats.data[0].data
+      .filter(rawData => SERIES_ORDER.indexOf(rawData.axis) > -1)
       .map(rawData => {
         if (timestamps.length !== rawData.values.length) {
           throw new Error('Invalid stats response');
         }
 
-        if (rawData.axis === 'count') {
+        if (rawData.axis === 'count()') {
```

Comment:
eventually, i'd like to be able to not have to do this for each timeseries so we can generate arbitrary charts, but going to leave this for now to keep the PR small